eng/CodeAnalysis.ruleset

```diff
@@ -24,9 +24,9 @@
     <Rule Id="CA1034" Action="None" />          <!-- Nested types should not be visible -->
     <Rule Id="CA1036" Action="None" />          <!-- Override methods on comparable types -->
     <Rule Id="CA1040" Action="None" />          <!-- Avoid empty interfaces -->
-    <Rule Id="CA1041" Action="None" />          <!-- Provide ObsoleteAttribute message -->
+    <Rule Id="CA1041" Action="Info" TemporarilyDisabled="true" />          <!-- Provide ObsoleteAttribute message -->
     <Rule Id="CA1043" Action="None" />          <!-- Use Integral Or String Argument For Indexers -->
-    <Rule Id="CA1044" Action="None" />          <!-- Properties should not be write only -->
+    <Rule Id="CA1044" Action="Info" TemporarilyDisabled="true" />          <!-- Properties should not be write only -->
```

Comment:
This rule is already violated by `IRunnableProjectConfig.SourceFile` and requires updating the public interface. We may consider not enabling this rule, or disable it for this member only.